docs/content/doc/help/faq.en-us.md

```diff
@@ -118,6 +118,12 @@ The correct path for the template(s) will be relative to the `CustomPath`
 2. If you are still unable to find a path, the default can be [calculated above](#where-does-gitea-store-what-file)
 3. Once you have figured out the correct custom path, you can refer to the [customizing Gitea]({{< relref "doc/administration/customizing-gitea.en-us.md" >}}) page to add your template to the correct location.
 
+## Does Gitea have a "GitHub/GitLab pages" feature?
+
+No, Gitea does not ship with that functionality and most likely never will.
```

Comment:
We should mention the reason.

Reply:
tldr: would be because of A. feature creep and B. that good external services witch integrate well exist

do you have any nice wording for that?

Reply:
The main reason I think is security. An internal page service cannot avoid csrf security problems.

Reply:
reworded